modules/tinymce/src/plugins/table/test/ts/browser/FakeSelectionTest.ts

```diff
@@ -139,7 +139,7 @@ describe('browser.tinymce.plugins.table.FakeSelectionTest', () => {
 
   it('TINY-7724: does not select CEF cell if contenteditable=true child is selected', () => {
     const editor = hook.editor();
-    editor.setContent('<table><tr><td contenteditable="false"><p contenteditable="true">1</p></td><td>2</td></tr><tr><td>3</td><td>4</td></tr></table>');
+    editor.setContent('<table><tbody><tr><td contenteditable="false"><p contenteditable="true">1</p></td><td>2</td></tr><tr><td>3</td><td>4</td></tr></tbody></table>');
```

Comment:
I added a tbody here since it was confusing to align the selection paths with browser normalization they always normalize the tables to have a tbody. I didn't do it for the rest of this file though seemed out of scope of this task.